README_utm_arm.txt

```diff
@@ -40,9 +40,11 @@ On an M1 Mac laptop, we cannot use virtual box, so follow these instructions ins
 
 
 3. Download and save the Ubuntu 20.04 ARM Server ISO
-   https://mac.getutm.app/gallery/ubuntu-20-04
+   https://mac.getutm.app/gallery/ubuntu-20-04 *
    https://cdimage.ubuntu.com/releases/20.04.4/release/
 
+   * click "Ubuntu Server for ARM", then click "Alternative and previous releases"
```

Comment:
Okay, so I see now, I never clicked the download link on the `mac.getutm.app` page and didn't realize it took you to Ubuntu's site.

Did you end up using any other part of that utm page other than the link into Ubuntu's site for the download? Going to "Alternative and previous releases" seems to only present amd64 releases and not the arm64 as we need.

Going to `https://cdimage.ubuntu.com/releases/20.04.4/release/` does give us the ARM server ISO pretty immediately.

My recommendation would be to rewrite this entire section to just have the one link and avoid the clickthrough from UTM's site:
```suggestion
   https://cdimage.ubuntu.com/releases/20.04/release/
```

and then we could include the `https://mac.getutm.app/gallery/ubuntu-20-04` at the top of this page as just a reference to it.

Reply:
I didn't download from the UTM site, so I agree we should just have people use the first link. I've updated the README accordingly.